.cicd/run.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+set -eo pipefail
+. ./.cicd/helpers/general.sh
+. ./$HELPERS_DIR/execute.sh
+
+execute mkdir -p $ROOT_DIR/build
+
+if [[ $(uname) == Darwin ]]; then
+    MAC_CMAKE="cmake -DCMAKE_BUILD_TYPE=Release .."
+    MAC_MAKE="make -j$JOBS"
+    MAC_TEST="ctest -j$JOBS --output-on-failure -T Test"
```

Comment:
These variables which contain the code which is to be executed are unnecessary complexity, especially given that they are only used twice (we can make it once, see below) and in the same file. Please remove.